pkg/enactmentstatus/conditions/conditions.go

```diff
@@ -123,6 +131,43 @@ func SetFailed(conditions *nmstate.ConditionList, reason nmstate.ConditionReason
 		reason,
 		"",
 	)
+	conditions.Set(
+		nmstate.NodeNetworkConfigurationEnactmentConditionAborted,
+		corev1.ConditionTrue,
+		reason,
+		message,
+	)
+}
+
+func SetConfigurationAborted(conditions *nmstate.ConditionList, message string) {
+	SetAborted(conditions, nmstate.NodeNetworkConfigurationEnactmentConditionConfigurationAborted, message)
+}
+
+func SetAborted(conditions *nmstate.ConditionList, reason nmstate.ConditionReason, message string) {
+	conditions.Set(
+		nmstate.NodeNetworkConfigurationEnactmentConditionFailing,
+		corev1.ConditionTrue,
```

Comment:
Abort is not Failing, failing means that applying the configuration didn't work out.

Reply:
You're right, thanks!